mrmustard/lab_dev/states/bargmann_eigenstate.py

```diff
@@ -28,23 +28,31 @@
 
 class BargmannEigenstate(Ket):
     r"""
-    The Bargmann eigenstate.
+    The `N`-mode Bargmann eigenstate.
 
-    .. code-block ::
+    Args:
+        modes: A list of modes.
+        alpha: The displacement of the state (i.e., the eigen-value).
+
+    Notes:
+        If ``alphas`` is a ``Sequence``, its length must be equal to `1` or `N`.
+        If its length is equal to `1`, all the modes share the same parameters.
+        Note that the only difference with ``Coherent(modes, alphas)`` is in its `c` parameter (and hence, does not have unit norm).
```

Comment:
this is no longer true: we don't support parallel gates, but we now have arbitrary batches :)